slycot/math.py

```diff
@@ -18,11 +18,106 @@
 #       MA 02110-1301, USA.
 
 from . import _wrapper
-from .exceptions import raise_if_slycot_error
+from .exceptions import SlycotParameterError, raise_if_slycot_error
```

Comment:
"Imported but unused". Please revert this. Technically the error is not raised from any code in this module. (The docstring does not count).